pkg/build/info.go

```diff
@@ -119,7 +117,7 @@ func (i *Info) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	var rawString string
 	err := unmarshal(&rawString)
 	if err == nil {
-		i.Name = rawString
+		i.Context = rawString
```

Comment:
should this be i.Image ?

Reply:
No, this is for the build unmarshaller on this scenario:
```yaml
services:
  api:
    build: api
```
We are passing the context. We were using name to later set the real context